package.json

```diff
@@ -59,12 +59,13 @@
     "query-string": "6.12.0",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
-    "react-hot-loader": "4.12.20",
+    "react-hot-loader": "^4.12.20",
+    "react-icons": "^3.9.0",
     "react-is": "16.13.1",
     "react-popper": "2.1.0",
     "react-router-dom": "5.1.2",
     "react-spinners": "^0.8.1",
-    "regenerator-runtime": "0.13.5",
+    "regenerator-runtime": "^0.13.5",
```

Comment:
I think we were specifically pegging this dependency to that version. Would you mind reverting this change if it wasn't necessary?